apps/web-giddh/src/app/audit-logs/components/sidebar-components/audit-logs.sidebar.component.ts

```diff
@@ -1,33 +1,33 @@
-import { of as observableOf, ReplaySubject } from 'rxjs';
-
-import { take, takeUntil } from 'rxjs/operators';
-import { LogsRequest } from '../../../models/api-models/Logs';
-import { UserDetails } from '../../../models/api-models/loginModels';
-import { CompanyResponse } from '../../../models/api-models/Company';
-import { CompanyService } from '../../../services/companyService.service';
-import { GroupService } from '../../../services/group.service';
-import { AccountService } from '../../../services/account.service';
-import { AppState } from '../../../store';
-import { GIDDH_DATE_FORMAT, GIDDH_DATE_FORMAT_UI } from '../../../shared/helpers/defaultDateFormat';
-import { Store } from '@ngrx/store';
-
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import {of as observableOf, ReplaySubject} from 'rxjs';
+
+import {map, take, takeUntil} from 'rxjs/operators';
+import {LogsRequest} from '../../../models/api-models/Logs';
+import {UserDetails} from '../../../models/api-models/loginModels';
+import {CompanyResponse} from '../../../models/api-models/Company';
+import {CompanyService} from '../../../services/companyService.service';
+import {GroupService} from '../../../services/group.service';
+import {AccountService} from '../../../services/account.service';
+import {AppState} from '../../../store';
+import {GIDDH_DATE_FORMAT, GIDDH_DATE_FORMAT_UI} from '../../../shared/helpers/defaultDateFormat';
+import {select, Store} from '@ngrx/store';
+
+import {Component, OnDestroy, OnInit} from '@angular/core';
 import * as moment from 'moment/moment';
-import { FormBuilder } from '@angular/forms';
-import { AuditLogsSidebarVM } from './Vm';
+import {FormBuilder} from '@angular/forms';
+import {AuditLogsSidebarVM} from './Vm';
 import * as _ from '../../../lodash-optimized';
-import { AuditLogsActions } from '../../../actions/audit-logs/audit-logs.actions';
-import { BsDatepickerConfig } from 'ngx-bootstrap/datepicker';
-import { IOption } from '../../../theme/ng-virtual-select/sh-options.interface';
+import {AuditLogsActions} from '../../../actions/audit-logs/audit-logs.actions';
+import {BsDatepickerConfig} from 'ngx-bootstrap/datepicker';
+import {IOption} from '../../../theme/ng-virtual-select/sh-options.interface';
 
 @Component({
     selector: 'audit-logs-sidebar',
     templateUrl: './audit-logs.sidebar.component.html',
     styles: [`
-    .ps {
-      overflow: visible !important
-    }
-  `]
+        .ps {
```

Comment:
Create a new scss for this and use here.

Reply:
sir this is only space change i didn't change any style in this component.